Chord: the app is Chord (name, bundle id). The data folder stays ~/Library/Application Support/Chromium.
--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,10 +1,10 @@
-COMPANY_FULLNAME=The Chromium Authors
-COMPANY_SHORTNAME=The Chromium Authors
-PRODUCT_FULLNAME=Chromium
-PRODUCT_SHORTNAME=Chromium
-PRODUCT_INSTALLER_FULLNAME=Chromium Installer
-PRODUCT_INSTALLER_SHORTNAME=Chromium Installer
+COMPANY_FULLNAME=The Chord Authors
+COMPANY_SHORTNAME=The Chord Authors
+PRODUCT_FULLNAME=Chord
+PRODUCT_SHORTNAME=Chord
+PRODUCT_INSTALLER_FULLNAME=Chord Installer
+PRODUCT_INSTALLER_SHORTNAME=Chord Installer
 COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Chromium Authors. All rights reserved.
-MAC_BUNDLE_ID=org.chromium.Chromium
+MAC_BUNDLE_ID=app.chord.Chord
 MAC_CREATOR_CODE=Cr24
 MAC_TEAM_ID=
